authorization-api-1_0.md

```diff
@@ -599,7 +602,184 @@ X-Request-ID: bfe9eb29-ab87-4ca3-be83-a1d5d8305720
 ~~~
 {: #example-search-response title="Example of an Search Response"}
 
+## Reverse-Search API
+The Access Reverse Search API does the reverse of the Search API: it enables a PEP or client to find out all the principals that can access a given asset.
```

Comment:
Instead of calling this "reverse search", should we use a terminology like "principal search" (in contrast with "asset search")?

Reply:
Responded in the issue #34

Reply:
Changed to "Subject search".